### ISSUE TITLE
[css21][css-inline] Meta bug for line height

### ISSUE THREAD

**Issue by frivoal** [w3c/csswg-drafts#1796]
This is the meta bug to track a series of issues based on the findings of https://github.com/w3c/csswg-drafts/issues/938#issuecomment-314690847.

The general premise is: css21 is partly ambiguous, partly self-contradictory about how height measurements of non replaced inline level boxes are supposed to happen. Implementations are not fully interoperable. The goal here is to find what is interoperable, to agree on what to do for things that are not, and to fix the specs accordingly.

**Note that I am only testing what happens with a single inline element, although I am using fallback fonts. Multiple inline elements on the line is a separate discussion.**

**Comment by frivoal** [comment]
Things all implementations agree about:
* https://github.com/w3c/csswg-drafts/issues/1797
* https://github.com/w3c/csswg-drafts/issues/1801
* https://github.com/w3c/csswg-drafts/issues/1802
* https://github.com/w3c/csswg-drafts/issues/1800
* https://github.com/w3c/csswg-drafts/issues/1804

Disagreements:
* https://github.com/w3c/csswg-drafts/issues/1798

TL;DR: We have interoperability in the general case both for the height of inline boxes, the position of the baseline therein, and for the height of the content area of inline boxes. Specs should be updated accordingly. The remaining cases where interop is lacking are odd corner cases, which we should harmonize and specify.

**Comment by frivoal** [comment]
@astearns @atanassov How do you want to manage/chair this? I am reasonably confident in my conclusions, but this should get meaningful review from others before we amend 2.1. I can write Pull Requests for spec changes and for tests to go with it, and these should get close scrutiny, but presumably we want (at least) some high level discussions prior to that.

I can try to present this during a teleconf, and see if we can get a resolution. If we don't, I'll whiteboard this at TPAC. If we do, I write the PRs, ask for review, and we revisit during TPAC, hopefully to resolve to merge them.

Pinging a few people who should probably review this sooner or later: @atanassov @FremyCompany @fantasai @dbaron @smfr @tabatkins @litherum

**Comment by kojiishi** [comment]
This is the one we talked about in Paris, correct? Can you point which one Blink behaves differently from other engines?

**Comment by frivoal** [comment]
@kojiishi Yes, it's that one. The difference is actually limited to a more narrow case than I had thought at first, but it is there. Details in #1798.

**Comment by kojiishi** [comment]
Ok, thanks, I'll take a look at #1798. Please let me know if there were other issues I should look at to see what Blink does.

**Comment by frivoal** [comment]
All sub issues have been closed, closing this one as well.
